.github/workflows/ci_workflow.yml

```diff
@@ -65,6 +65,8 @@ jobs:
             build-type: Release
             build-shared: 'ON'
             build-docs: 'ON'
+            build-gpu: 'ON'
+            use-headless: 'ON'
```

Comment:
Based on the new approach, you must now have all `build-gpu`& `use-headless` be disabled.